Banyan/Manifest.toml

```diff
@@ -303,6 +313,18 @@ git-tree-sha1 = "5d494bc6e85c4c9b626ee0cab05daa4085486ab1"
 uuid = "5ced341a-0733-55b8-9ab6-a4889d929147"
 version = "1.9.3+0"
 
+[[MPI]]
```

Comment:
Just noting that these changes are untested. We need to ensure that we are properly able to install/configure MPI on the cluster given these Project/Manifest toml files.